tests/unit/components/NcAvatar/NcAvatar.spec.ts

```diff
@@ -42,7 +42,7 @@ describe('NcAvatar.vue', () => {
 		await nextTick()
 
 		expect(wrapper.find('.avatardiv__user-status').exists()).toBe(true)
-		expect(wrapper.attributes('aria-label')).toBe('Avatar of J. Doe, online')
+		expect(wrapper.find('.action-item__menutoggle').attributes('aria-label')).toBe('Avatar of J. Doe, online')
```

Comment:
still not a fan of testing using classes (better semantially like `button[aria-label]` or similar).
But the tests already do this so no blame just my 5 cents